GitCommands/UserRepositoryHistory/RecentRepoInfo.cs

```diff
@@ -77,11 +106,21 @@ public void SplitRecentRepos(IList<Repository> recentRepositories, List<RecentRe
 
             // the maxRecentRepositories repositories will be added at beginning
             // rest will be added in alphabetical order
+
+            var ourSubmodules = submoduleInfoResult?.OurSubmodules.ToDictionary(module => module.Path);
+            var worktrees = worktreeInfoResult?.Worktrees.ToDictionary(worktree => worktree.Path);
+
             foreach (Repository repository in recentRepositories)
             {
-                bool mostRecent = (mostRecentRepos.Count < n && repository.Anchor == Repository.RepositoryAnchor.None) ||
+                bool mostRecent =
+                    (mostRecentRepos.Count < n && repository.Anchor == Repository.RepositoryAnchor.None) ||
```

Comment:
Why changed?

Reply:
Rider changed it because this line exceeds the maximum line length (120).